vignettes/ospsuite.Rmd

```diff
@@ -11,96 +11,339 @@ vignette: >
+- Visualizing simulation results
+- Understanding the basic ospsuite workflow
 
-The workflow steps are described in the following vignettes:
+All examples in this tutorial and the rest of the documentation use the Aciclovir example model (unless stated otherwise), which is included in the package and can be accessed with:
 
-* [Loading a simulation and accessing entities](load-get.html)
-* [Changing parameter and molecule start values](set-values.html)
-* [Running a simulation](run-simulation.html)
-* [Efficient calculations](efficient-calculations.html)
-* [Creating individuals](create-individual.html)
-* [Population simulations](create-run-population.html)
-* [PK Analysis](pk-analysis.html)
-* [Sensitivity analysis](sensitivity-analysis.html)
-* [Table parameters](table-parameters.html)
-* [Dimensions and Units](unit-conversion.html)
-* [Working with data sets and import from excel](observed-data.html)
-* [Working with `DataCombined` class](data-combined.html)
-* [Visualizations with `DataCombined`](data-combined-plotting.html)
+```r
+system.file("extdata", "Aciclovir.pkml", package = "ospsuite")
+```
+
+
+## Quick Start: Basic Simulation
+
+This example demonstrates the essential ospsuite workflow. Detailed explanations follow in subsequent sections.
+
+```{r, echo=TRUE, results='hide'}
+# Load the ospsuite package
+library(ospsuite)
+
+# Load the built-in example simulation
+simFilePath <- system.file("extdata", "Aciclovir.pkml", package = "ospsuite")
+sim <- loadSimulation(simFilePath)
+
+# Run the simulation
+results <- runSimulations(sim)
+
+# Create a quick visualization
+myDataCombined <- DataCombined$new()
+myDataCombined$addSimulationResults(results[[1]])
+plotIndividualTimeProfile(dataCombined = myDataCombined)
+```
+
+This example loads, runs, and visualizes a PBPK simulation. The plot shows the concentration-time profile for Aciclovir in plasma after IV administration.
+
+## Step-by-Step Tutorial
+
+The following sections detail each workflow step and demonstrate ospsuite capabilities.
+
+### Loading Simulations
+
+Simulations require a **.pkml** format file (exported from PK-Sim or MoBi):
+
+```{r, echo=TRUE, message=TRUE}
+# Using the included example
+simFilePath <- system.file("extdata", "Aciclovir.pkml", package = "ospsuite")
+sim <- loadSimulation(simFilePath)
+
+# For custom files, use:
+# sim <- loadSimulation("path/to/simulation.pkml")
+
+# Explore the simulation
+print(sim)
+```
+
+### Exploring Model Structure
+
+After loading, explore the simulation structure and parameters:
+
+```{r, echo=TRUE}
+# Get the simulation tree structure
+simTree <- getSimulationTree(sim)
+
+# Explore parameter paths
+simTree$Organism$Weight$path
+
+# Find parameters using wildcards
+getAllParametersMatching("**|Dose*", sim)
+
+# Or search for specific terms
+grep("Dose", getAllParameterPathsIn(container = sim), value = TRUE)
+```
+
+**Note:** Parameter paths in ospsuite match those displayed in PK-Sim or MoBi.
+
```

Comment:
Here you could add the hint starting with MoBi 12.0, parameter paths can be copied from simulations via
<img width="382" height="242" alt="grafik" src="https://github.com/user-attachments/assets/101f5c0c-0b3d-4dbd-9758-c073f12d8366" />
and container/observer paths as well
<img width="305" height="337" alt="grafik" src="https://github.com/user-attachments/assets/38ddac11-581b-4a19-99a3-0ed1b615bbf9" />